src/app/app.component.html

```diff
@@ -1,3 +1,5 @@
+<div class="fix-container">
```

Comment:
@SMahil you can take this up, create a new issue and work on it.
Also, we are not going to focus on mobile.